examples/howto/js_events.py

```diff
@@ -53,7 +53,7 @@ def display_event(div, attributes=[]):
 # Add a div to display events and a button to trigger button click events
 
 div = Div(width=1000)
-button = Button(label="Button", button_type="success")
+button = Button(label="Button", button_type="success", width=300)
 layout = column(button, row(p, div))
```

Comment:
I think all the cases are the same issue, so I will answer them here. The layout works correctly by fitting `button` into the available space, which is ~1600px total. If you want to have a button of specific width, then setting `width` will fix it at the given value. If the intent is to align the button with the plot, then I would change this example altogether:
```py
div = Div()
button = Button(label="Button", button_type="success")
layout = row(column(button, p), div)
```
Note that `div` will be initially of zero width, but adding content will trigger relayout.